DevTools/UserspaceEmulator/main.cpp

```diff
@@ -39,12 +39,11 @@
 int main(int argc, char** argv, char** env)
 {
     if (argc == 1) {
-        out() << "usage: UserspaceEmulator <command>";
+        out() << "Usage: UserspaceEmulator <command>";
         return 0;
     }
 
-    // FIXME: Allow specifying any command in $PATH instead of requiring a full executable path.
-    const char* executable_path = argv[1];
+    auto executable_path = Core::find_executable_in_path(argv[1]);
```

Comment:
This seems to not work:

```
$ ue ls
open: Bad address
Unable to map
```

(But `ue /bin/ls` works fine.)

Reply:
This is weird, I could swear that I tried to run `UserspaceEmulator id` and it worked. But somehow it's broken.

Reply:
#3514